response_operations_ui/views/case.py

```diff
@@ -31,18 +34,27 @@ def get_response_statuses(ru_ref, error=None):
 
     case_groups = case_controller.get_case_groups_by_business_party_id(reporting_unit['id'])
     case_group = case_controller.get_case_group_by_collection_exercise(case_groups, exercise['id'])
-
-    return render_template('change-response-status.html',
+    case_group_status = case_group['caseGroupStatus']
+    case_id = get_case_by_case_group_id(case_group['id']).get('id')
+    is_complete = False
```

Comment:
We could create the `is_complete` and `completed_timestamp` objects in a slightly more functional way
```
is_complete = case_group_status in COMPLETE_STATE
completed_timestamp = get_timestamp_for_completed_case_event(case_id) if is_complete else None
```
Then you don't need to initialise them and then change them